src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Box, Button, GU, TextInput, Info, IconMail } from '@aragon/ui'
+import { login } from './notification-service-api'
+import { getEthNetworkType } from '../../../local-settings'
```

Comment:
Ok. I just realised that we have a small inconsistency with the backend which uses `mainnet` rather than `main` as the constant (https://github.com/aragonone/notification-service/blob/575dc13f20dd0555112a7df07b257678ef55ebea/lib/constants.js#L11).